test/integration/traversal/TraversalTest.java

```diff
@@ -186,7 +186,7 @@ public void mixed_variable_and_edge_role_players() {
             proc.forwardRelating(_0, role_inst);
             proc.backwardPlaying(role_inst, x);
             proc.forwardIsa(role_inst, role, true);
-            proc.forwardRolePlayer(_0, e, set(Label.of("employer", "employment")));
+            proc.forwardRolePlayer(_0, e, 0, set(Label.of("employer", "employment")));
```

Comment:
* rolePlayer edge now requires `repetition` number to construct to differentiate parallel edges